lib/ansible/modules/packaging/os/yum_repository.py

```diff
@@ -74,7 +74,7 @@
         not used. Use C(0) to turn off delta rpm processing. Local repositories
         (with file:// I(baseurl)) have delta rpms turned off by default.
   description:
-    required: false
+    required: true
```

Comment:
It's only required when state is 'present' though. This is made note of in the [option description](https://github.com/ansible/ansible/pull/31700/files#diff-3e0703568828cb54958e276ae5273b3aL81) as the [docs recommend](http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-block).

Reply:
Thx for info, my mistake. Closing...